src/CfgMgr32/CfgMgr32.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright © .NET Foundation and Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Exported functions from the CfgMgr32.dll Windows library
+    /// that are available to Desktop apps only.
+    /// </content>
+    [OfferFriendlyOverloads]
+    public static partial class CfgMgr32
+    {
+        private const int MAX_DEVICE_ID_LEN = 200;
+
+        /// <summary>
+        /// The callback routine invoked by the Configuration Manager when listening for events.
+        /// </summary>
+        /// <param name="notify">
+        /// The handle of the notification context which invoked the callback.
+        /// </param>
+        /// <param name="context">
+        /// A user-provided callback context.
+        /// </param>
+        /// <param name="action">
+        /// The device action which triggered the callback.
+        /// </param>
+        /// <param name="eventData">
+        /// The callback data.
+        /// </param>
+        /// <param name="eventDataSize">
+        /// The size of the callback data struct.
+        /// </param>
+        /// <returns>
+        /// If responding to a <see cref="CM_NOTIFY_ACTION.CM_NOTIFY_ACTION_DEVICEQUERYREMOVE"/> notification,
+        /// the <see cref="CM_NOTIFY_CALLBACK"/> callback should return either <see cref="Win32ErrorCode.ERROR_SUCCESS"/> or <see cref="Win32ErrorCode.ERROR_CANCELLED"/>,
+        /// as appropriate. Otherwise, the callback should return <see cref="Win32ErrorCode.ERROR_SUCCESS"/>.
+        /// The callback should not return any other values.
+        /// </returns>
+        /// <seealso href="https://docs.microsoft.com/en-us/windows/win32/api/cfgmgr32/nf-cfgmgr32-cm_register_notification#remarks"/>
+        public unsafe delegate Win32ErrorCode CM_NOTIFY_CALLBACK(
+            IntPtr notify, // This is a SafeNotificationHandle, but safe handles cannot be marshalled
```

Comment:
Good comment, but it is inaccessible to users. Can you add the part about this coming from a `SafeNotificationHandle` to the parameter doc?